sotodlib/toast/sim_hwpss.py

```diff
@@ -43,8 +44,19 @@ def exec(self, data):
         for obs in data.obs:
             tod = obs["tod"]
             focalplane = obs["focalplane"]
+            
             # Get HWP angle
             chi = tod.local_hwp_angle()
+            
+            # Get times
+            times = tod.local_times()
+            
+            # Fast sample HWP angle with a factor 100
+            fast_time = np.linspace(times[0], times[-1], 1+(len(times)-1)*100)
```

Comment:
This can be a significant increase in memory consumption. Do we absolutely need to oversample 100X or would, say, 10X do the job as well?

Reply:
This is also no longer required in the latest fix.